edge_computing/image_based_upgrade/ztp-image-based-upgrade.adoc

```diff
@@ -37,6 +37,8 @@ include::modules/ztp-image-based-upgrade-upgrade.adoc[leveloffset=+1]
 
 * xref:../../edge_computing/image_based_upgrade/ztp-image-based-upgrade.adoc#ztp-image-based-upgrade-rollback_ztp-gitops[Moving to the Rollback stage of the image-based upgrade with {lcao} and {ztp}]
 
+* xref:../../edge_computing/cnf-talm-for-cluster-upgrades.adoc#talo-policies-concept_cnf-topology-aware-lifecycle-manager[Update policies on managed clusters]
```

Comment:
Outside the updates for this PR but somewhat related, but maybe remove "section" so it aligns with the Deleting Operators...sentence style: 

![Screenshot from 2024-07-31 11-38-08](https://github.com/user-attachments/assets/b97125dc-0b13-4e73-9e6d-34e7e903f34e)